verifiable-db/src/revelation/api.rs

```diff
@@ -126,121 +290,168 @@ impl<
         preprocessing_proof: Vec<u8>,
         query_bounds: &QueryBounds,
         placeholders: &Placeholders,
-        placeholder_hash_ids: [PlaceholderId; NUM_PLACEHOLDERS_HASHED],
+        predicate_operations: &[BasicOperation],
+        results_structure: &ResultStructure,
```

Comment:
Please fix the out-of-date docstring.

Reply:
Done in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/373/commits/3644f30ea00e92324ebd07c84c164757a945fbb1